core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -311,7 +311,7 @@ class CityInfo {
     fun getWorkableTiles() = tilesInRange.asSequence().filter { it.getOwner() == civInfo }
     fun isWorked(tileInfo: TileInfo) = workedTiles.contains(tileInfo.position)
 
-    fun isCapital(): Boolean = cityConstructions.builtBuildings.contains(capitalCityIndicator())
+    fun isCapital(): Boolean = cityConstructions.builtBuildings.any { getRuleset().buildings[it]!!.hasUnique(UniqueType.IndicatesCapital) }
```

Comment:
Debug fluke? But shows a few things: `capitalCityIndicator` would be specific to civ but lives in city, could use a cache, and is buggy: It could choose another civ's unique palace when 'your' civ has no unique palace.